githubapp/ldap.py

```diff
@@ -47,7 +50,11 @@ def get_group_members(self, group_name):
             if entry['type'] == 'searchResEntry':
                 for member in entry['attributes'][self.LDAP_GROUP_MEMBER_ATTRIBUTE]:
                     try:
-                        member_list.append(self.get_attr_by_dn(member))
+                        if 'uid' in member:
```

Comment:
```suggestion
                        if any(attr in member.casefold() for attr in ['uid', 'cn'])::
```

I haven't tested this with OpenLDAP, but it works with AD now